### DISCUSSION TITLE
Known bugs before 2.8

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#7012] [General]
I popped open the app to see what polish was needed before 2.8… 

![image](https://github.com/StoDevX/AAO-React-Native/assets/464441/9aa74daa-d224-4076-a61e-becd9cff0d10)

Did we know that:

- menus has a grey background
- list items have no highlight
- navbar and tabbar both are using the default blue/white colors

I hope to save some time this weekend to look at some of these…

**Comment by drewvolz** [comment]
Here's the 2.8 milestone https://github.com/StoDevX/AAO-React-Native/milestone/20

Here is the issue tracking 2.8 https://github.com/StoDevX/AAO-React-Native/issues/4774

**Reply by hawkrives** [reply]
Related issue for some colors: https://github.com/StoDevX/AAO-React-Native/issues/3168#issuecomment-1140560714

**Reply by drewvolz** [reply]
Closed the above issue with Android support deprecated and those colors looking fine on iOS
